chart/epinio/questions.yml

```diff
@@ -0,0 +1,128 @@
+questions:
+- variable: email
+  label: Email
+  description: "Email to use for getting notifications about your certificates"
+  default: "epinio@suse.com"
+  type: strings
+  required: false
+  group: "General settings"
+- variable: tlsIssuer
+  label: Tls issuer
+  description: "Name of the cluster issue to use"
+  default: "epinio-ca"
+  type: enum
+  required: false
+  group: "General settings"
+  options:
+    - "epinio-ca"
+    - "selfsigned-issuer"
+    - "letsencrypt-production"
+- variable: user
+  label: API username
+  description: "The user name for authenticating all API requests"
+  type: strings
+  required: false
+  group: "General settings"
+- variable: password
+  label: API password
+  description: "The password for authenticating all API requests"
+  type: password
+  required: false
+  group: "General settings"
+- variable: domain
+  label: Domain
+  description: "Domain for the application"
+  type: strings
+  required: false
+  group: "General settings"
+- variable: accessControlAllowOrigin
+  label: Access control allow organization
+  description: "Domain allowed to access the API. Comma-separated."
```

Comment:
- `Domain allowed to access the API` - this is true, however it would be good to ensure the user knows they must configure this for the UI to work. I think the questions framework may support tooltips, if so could we add something like `Should match the domain which servers the UI`?
- `Comma-separated` - This isn't quite supported, I think Epinio places this value directly into the response header - https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin#directives

Reply:
Regarding tooltips, I don't see anything about it in the documentation https://rancher.com/docs/rancher/v2.6/en/helm-charts/creating-apps/#additional-files-for-rancher-charts
But, I can simply improve the description field.

Reply:
Ahh, my mistake, they're not there (see `rancher/dashboard` `components/Questions/String.vue`). 

How about something like `Domain which serves the UI (to access the API)`?

Reply:
I'm ok for `Domain which serves the Rancher UI (to access the API)` if you agree too.

Reply:
Looking good! Can't comment on the rest but this bit is fine